packages/repository/test/acceptance/has-many.relation.acceptance.ts

```diff
@@ -59,6 +55,43 @@ describe('HasMany relation', () => {
     expect(persisted.toObject()).to.deepEqual(order.toObject());
   });
 
+  it('can patch many instances', async () => {
+    await createSamples();
+    await testPatch();
+
+    async function testPatch() {
+      const patchObject = {description: 'new order'};
+      const arePatched = await customerOrders.patch(patchObject);
+      // tslint:disable-next-line: no-unused-expression
+      expect(arePatched).to.equal(2);
+      const patchedItems = await customerOrders.find();
+      expect(patchedItems).to.have.length(2);
+      patchedItems.forEach(order => {
```

Comment:
Executing the same assertion multiple times in a loop usually produces unhelpful test failure messages. My proposal:

```ts
const actualData = _.pick(patchedItems, ['customerId', 'description']);
expect(actualData).to.eql([
  {customerId: existingCustomerId, description: 'new order'},
  {customerId: existingCustomerId, description: 'new order'},
]);
```

Also please verify the following aspects of the `patch` operation:
 - The operation touched only the properties specified in the data (i.e. `description`) and no other properties were changed (e.g. `isDelivered`).
 - What happens when the data is trying to change the relation key (`customerId`) - do we silently ignore that value or reject the request with an error? I personally prefer the error to tell the clients they are doing something wrong.